### PR TITLE
docs: update stories featuring bootstrap

### PR DESCRIPTION
**Pull request by ahasall**
Bootstrap 4 has been released.
`npm install bootstrap --save` installs Bootstrap 4 instead of Bootstrap 3.
To install Boostrap 3, we should do `npm install bootsrap@3`

### CODE REVIEW

**Comment by ahasall** [comment]
I signed it!

File: docs/documentation/stories/include-bootstrap.md [hunk 1]
```diff
@@ -22,11 +22,11 @@ With the new project created and ready you will next need to install bootstrap t
 Using the `--save` option the dependency will be saved in package.json
 
 ```sh
-# version 3.x
```
Comment (markgoho) [inline docs/documentation/stories/include-bootstrap.md]: Do we typically add instructions for adding old packages to projects? I'd suggest we remove the v.3 installation notes altogether.

Comment (ahasall) [reply docs/documentation/stories/include-bootstrap.md]: Yes. You are right. I updated the PR.

File: docs/documentation/stories/global-lib.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ First install Bootstrap from `npm`:
 ```bash
 npm install jquery --save
 npm install popper.js --save
```
Comment (rafaelss95) [inline docs/documentation/stories/global-lib.md]: It's not something you added, but can we add that neither `jquery` nor `popper.js` are needed if you pretend to use only CSS? With the current docs people tend to feel that they have to install them obligatory.

Comment (ahasall) [reply docs/documentation/stories/global-lib.md]: I totally agree with you. I will update my PR accordingly.

Comment (ahasall) [reply docs/documentation/stories/global-lib.md]: I updated the PR. Tell me what you think about the changes.

**Comment by ahasall** [comment]
@markgoho or @rafaelss95,
Is there any action I can take to improve this PR so that it can be merged ?
Thanks

**Comment by filipesilva** [comment]
Heya, this pr is superseded by https://github.com/angular/angular-cli/pull/10541.

**Pull request closed without merging**